lib/spack/spack/cmd/modules/__init__.py

```diff
@@ -39,7 +39,7 @@ def setup_parser(subparser):
         action='store_true'
     )
     arguments.add_common_arguments(
-        refresh_parser, ['constraint', 'yes_to_all']
+        refresh_parser, ['installed_spec', 'yes_to_all']
```

Comment:
@alalazo my refactoring broke all of the commands that use your `constraint` positional arg. For now, I think I'm going to revert any changes to `spack find` and `spack module`, but you may want to merge these with `installed_spec` someday.